src/main/java/main/Duke.java

```diff
@@ -0,0 +1,57 @@
+package main;
```

Comment:
Your code in general is readable and clear! It is easy to follow. Great job on it!